docs/next.config.js

```diff
@@ -6,4 +6,14 @@ module.exports = {
     appDir: true,
     externalDir: true,
   },
+  images: {
```

Comment:
Maybe not for solving now as this is likely part of a cloud-image refactor but using `next/image` out of the box with `keystatic.io/images` feels like a clash of technologies. i.e. two competing image optimisation services + CDNs. We could turn off Next's image optimisation here with [unoptimized](https://nextjs.org/docs/app/api-reference/components/image#unoptimized) or a [custom loader](https://nextjs.org/docs/app/api-reference/next-config-js/images). Or maybe it's fine taking the path we have with creating our own image component.

Reply:
Yeah I switched back and forward with this one, but you are right we shouldn't be using next/image here. I'll replace with our own images component.

Reply:
Cool cool. There's more to solve WRT layout shift and figuring out + providing height/width to our cloud-image's `img` element. It would be nice to have a clear picture of how `next/image` could be best used by other folks picking up keystatic cloud as well, if they were keen to stick with next/image. Since we control storage, CDN and optimising, rather than just optimising, to me it would make sense to streamline the whole flow. Anywho, just thinking out load ATM 😊